test/View/RouteNotFoundStrategyTest.php

```diff
@@ -24,21 +24,29 @@
 use Laminas\Mvc\MvcEvent;
 use PHPUnit\Framework\TestCase;
 use Prophecy\Argument;
+use Prophecy\PhpUnit\ProphecyTrait;
 use ReflectionClass;
 use ReflectionMethod;
 
 class RouteNotFoundStrategyTest extends TestCase
 {
     use EventListenerIntrospectionTrait;
+    use ProphecyTrait;
 
     /**
      * @var RouteNotFoundStrategy
      */
     protected $strategy;
 
-    public function setUp()
+    public function setUp() : void
     {
-        $this->strategy = new RouteNotFoundStrategy();
+        $this->strategy = new class extends RouteNotFoundStrategy
```

Comment:
This test stubs parts of the system under test: we should use the `RouteNotFoundStrategy` as-is

Reply:
updated.